applications/sdp/mspi/src/hrt/hrt.c

```diff
@@ -363,46 +402,15 @@ void hrt_read(volatile hrt_xfer_t *hrt_xfer_params)
 					  BYTE_3_SHIFT);
 		}
 	} else {
-		uint32_t i = 0;
+		apply_shiftcnt(hrt_xfer_params, &rx_shift_ctrl, 0);
+		workaround_two_pulses(hrt_xfer_params, &rx_shift_ctrl);
 
+		uint32_t i = 1;
 		for (; i < hrt_xfer_params->xfer_data[HRT_FE_DATA].word_count; i++) {
```

Comment:
I do not see `i` being used anywhere outside of this loop, so these lines can be merged:
```suggestion
		for (uint32_t i = 1; i < hrt_xfer_params->xfer_data[HRT_FE_DATA].word_count; i++) {
```